tests/test_js_code.py

```diff
@@ -0,0 +1,52 @@
+import pytest
```

Comment:
I think we should not have a generic `test_jc_code.py` file, as it does not tell you what gets tested.
Let's keep the use-case-specific naming, as it doesn't matter if the test uses cypress or pytest only to test a feature.